phpstan.neon.dist

```diff
@@ -5,3 +5,9 @@ parameters:
     paths:
         - ./src
         - ./tests
+    ignoreErrors:
+        # That class no longer holds methods in DBAL 4, but needs to be called with DBAL 3
+        - message: '#Call to an undefined static method Doctrine\\DBAL\\Types\\ConversionException::.*#'
```

Comment:
@greg0ire this will throw an error when the method *does* exists (i,e in `DBAL v4`)

Reply:
You're right. Either set `reportUnmatchedErrors` to false, or use different configs for different jobs, or run static analysis with only one version of DBAL.

Reply:
Just did that ;)

Thanks!